src/lib/storage/reference_column.hpp

```diff
@@ -113,6 +113,8 @@ class ReferenceColumn : public BaseColumn {
 
   std::shared_ptr<BaseColumn> copy_using_allocator(const PolymorphicAllocator<size_t>& alloc) const override;
 
+  MemoryUsage estimate_memory_usage(MemoryUsageEstimationMode estimation_mode) const override;
```

Comment:
`ValueColumn::estimate_memory_usage` has a default argument. It should consistent.

Reply:
I concur with the arguing that when specifying the defargs for overrides, you risk deviating from the defargs of the base function and thus should omit them http://www.codingstandard.com/rule/9-1-2-make-default-arguments-the-same-or-absent-when-overriding-a-virtual-function/

Reply:
Okay, my bad. I though `ValueColumn` had a default argument.